### DISCUSSION TITLE
Stuck at creating. First node: Waiting for all the other servers... Other nodes: Waiting for the first server...

### DISCUSSION THREAD

**Discussion by ajunca** [minio/minio#13021] [Q&A]
## Expected Behavior
Minio cluster is created successfully

## Current Behavior
It just gets stuck at creation. 

First node message:
Waiting for all other servers to be online to format the disks.

Other nodes:
Waiting for the first server to format the disks.

## Steps to Reproduce (for bugs)
Created 4 VM's with Ubuntu Server 20.04 with 2TB disk attached at each. Configured everything and downloaded latest minio binary on each. I tried multiple versions of the command to start up the server (also with TLS and without). Some examples of what I tried (http version of all them tested):

- MINIO_ROOT_USER=admin MINIO_ROOT_PASSWORD=password ./minio server --console-address ":9001" https://192.168.100.4{1...4}/mnt/data1
- MINIO_ROOT_USER=admin MINIO_ROOT_PASSWORD=password ./minio server --console-address ":9001" https://192.168.100.4{1...4}:9000/mnt/data1
- MINIO_ROOT_USER=admin MINIO_ROOT_PASSWORD=password ./minio server --console-address ":9001" https://node{1...4}.s3.cluster.local.miceli.tech/mnt/data1
- MINIO_ROOT_USER=admin MINIO_ROOT_PASSWORD=password ./minio server --console-address ":9001" https://node{1...4}.s3.cluster.local.miceli.tech:9000/mnt/data1

## Your Environment
All four nodes are proxmox VM's running Ubuntu Server 20.04. No firewalls. Running lastest minio binary from [https://min.io/download](https://min.io/download)

**Comment by harshavardhana** [comment]
@junky89 we run the same binary in 1000's of instances -are you sure your networking is setup correctly? 

Also make sure your process has write access to the drives.

**Comment by harshavardhana** [comment]
Since this is not an issue moved it to a discussion..
